rockcraft/templates/test/spread.yaml.j2

```diff
@@ -0,0 +1,35 @@
+project: {{ name }}
+
+backends:
+  craft:
+    type: craft
+    systems:
+      - ubuntu-24.04:
+      #- ubuntu-22.04:
+
+suites:
+  spread/general/:
+    summary: General integration tests
+
+prepare: |
+  sudo snap install docker
+
+  # For 'rockcraft.skopeo'
+  sudo snap install --classic rockcraft
+
+  # Wait for docker daemon to come online
+  sudo apt install --yes retry
+  retry --times=10 --delay 2 -- docker run hello-world
+  sudo apt remove --yes retry
+
+  # Load the rock into docker
+  ROCK=$(ls $PROJECT_PATH/{{ name }}*.rock)
```

Comment:
this branch doesn't have craft-application 5.2, so `CRAFT_ARTIFACT` will come in a later PR

Reply:
@cmatsuoka I tried to update this to use `CRAFT_ARTIFACT`, like this:
```
  # Load the rock into docker
  sudo rockcraft.skopeo --insecure-policy copy "oci-archive:$CRAFT_ARTIFACT" docker-daemon:craft-test:latest
```

... but got this `prepare` failure:

```
:: + sudo rockcraft.skopeo --insecure-policy copy oci-archive:/root/proj//root/project/spread/craft-test_0.1_amd64.rock docker-daemon:craft-test:latest
:: time="2025-04-29T18:24:09Z" level=fatal msg="Invalid source name oci-archive:/root/proj//root/project/spread/craft-test_0.1_amd64.rock: lstat /root/proj/root: no such file or directory"
```

is this expected/known?

Reply:
As discussed in the meeting, we'll need to fix this in craft application because the `pack` method returns file names in snapcraft and full paths in other applications. A workaround would be to return the filename here too.